config/default_bpnode_fcncodes.h

```diff
@@ -0,0 +1,41 @@
+/************************************************************************
```

Comment:
This comment applies to all new source files.  I just want to verify that the Docket No is valid for GSFC BP project.  It' appears to be re-used from "core Flight System: Bootes" currently

Reply:
@dzbaker - do you know what we should be putting at the top of our files?

Reply:
Yeah this was just from sample_app. It's probably not valid but idk what it should be instead

Reply:
For now, let's put this:

/*
 * NASA Docket No. GSC-18,587-1 and identified as “The Bundle Protocol Core Flight
 * System Application (BP) v6.5”
 *
 * Copyright © 2020 United States Government as represented by the Administrator of
 * the National Aeronautics and Space Administration. All Rights Reserved.
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *
 */

Reply:
Done